src/app/components/category/CategoryCard.tsx

```diff
@@ -12,9 +12,12 @@ const CategoryCard: React.FC<Props> = ({ image, label }) => {
   return (
     <div className="w-[363px] h-[417px] mx-[12.5px] relative z-10 font-lato">
       <img src={image} alt="" width={363} height={417} />
-      <div className="flex items-center justify-center h-[60px] font-extrabold text-lg bg-white text-dark-orange absolute bottom-[22px] left-[23px] right-[23px] mx-auto">
+      <a
+        href="#"
+        className="flex items-center justify-center h-[60px] font-extrabold text-lg bg-white text-dark-orange absolute bottom-[22px] left-[23px] right-[23px] mx-auto"
```

Comment:
This class is so long so for code readability please use the apply directives from here and refactor https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply